modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2726,6 +2801,35 @@ private void _assertObjectEntriesSize(long size) throws Exception {
 			objectEntries.toString(), size, objectEntries.size());
 	}
 
+	private void _assertObjectEntryProperties(
+			Timestamp expectedDate, BigDecimal expectedBigDecimal,
+			double expectedDecimal, int expectedInteger,
+			long expectedLongInteger, long objectEntryId,
+			String expectedRichText, String expectedText)
+		throws Exception {
+
+		ObjectEntry objectEntry = _defaultObjectEntryManager.getObjectEntry(
+			_simpleDTOConverterContext, _objectDefinition2, objectEntryId);
+
+		Map<String, Object> properties = objectEntry.getProperties();
+
+		Assert.assertEquals(
+			expectedDate, properties.get("dateObjectFieldName"));
+		Assert.assertEquals(
+			expectedDecimal, properties.get("decimalObjectFieldName"));
+		Assert.assertEquals(
+			expectedInteger, properties.get("integerObjectFieldName"));
+		Assert.assertEquals(
+			expectedLongInteger, properties.get("longIntegerObjectFieldName"));
+		Assert.assertEquals(
+			expectedBigDecimal,
+			properties.get("precisionDecimalObjectFieldName"));
+		Assert.assertEquals(
+			expectedRichText, properties.get("richTextObjectFieldName"));
+		Assert.assertEquals(
+			expectedText, properties.get("textObjectFieldName"));
+	}
```

Comment:
@pedro-oliveira446 can we use the method `_assertEquals` instead of create a new one?